### ISSUE TITLE
short import

### ISSUE THREAD

**Issue by Cologler** [sixty-north/asq#14]
Use `from asq import query` instead of `from asq.initiators import query`.

I read the readme.md every time when I try to use this library. 

So sad.

**Comment by scls19fr** [comment]
We just need in `__init__.py`

```python
from .initiators import query  # noqa
```

and in `test_asq.py` change
```python
from asq.initiators import query
```

to

```python
from asq import query
```

**Comment by rob-smallshire** [comment]
Addressed by 9e1224689c516e2c5de0e9337f9590df11b4f869
